[2022-08-06 19:36:57,101] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-10-28T00:00:00+00:00 [queued]>
[2022-08-06 19:36:57,118] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-10-28T00:00:00+00:00 [queued]>
[2022-08-06 19:36:57,118] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:36:57,118] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 19:36:57,118] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:36:57,138] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-10-28 00:00:00+00:00
[2022-08-06 19:36:57,146] {standard_task_runner.py:52} INFO - Started process 957 to run task
[2022-08-06 19:36:57,149] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-10-28T00:00:00+00:00', '--job-id', '278', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmph66vkrhg', '--error-file', '/tmp/tmp_51054vx']
[2022-08-06 19:36:57,149] {standard_task_runner.py:80} INFO - Job 278: Subtask second_task
[2022-08-06 19:36:57,250] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-10-28T00:00:00+00:00 [running]> on host b29c168c1666
[2022-08-06 19:36:57,361] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-10-28T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-28T00:00:00+00:00
[2022-08-06 19:36:57,361] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 19:36:57,362] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 19:36:57,378] {subprocess.py:85} INFO - Output:
[2022-08-06 19:36:57,380] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 19:36:57,380] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 19:36:57,467] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211028T000000, start_date=20220806T193657, end_date=20220806T193657
[2022-08-06 19:36:57,530] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 19:36:57,587] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:50:49,190] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-10-28T00:00:00+00:00 [queued]>
[2022-08-06 21:50:49,207] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-10-28T00:00:00+00:00 [queued]>
[2022-08-06 21:50:49,207] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:50:49,207] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:50:49,207] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:50:49,224] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-10-28 00:00:00+00:00
[2022-08-06 21:50:49,231] {standard_task_runner.py:52} INFO - Started process 1137 to run task
[2022-08-06 21:50:49,234] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-10-28T00:00:00+00:00', '--job-id', '276', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpepjuww7w', '--error-file', '/tmp/tmp6p2m5v7x']
[2022-08-06 21:50:49,235] {standard_task_runner.py:80} INFO - Job 276: Subtask second_task
[2022-08-06 21:50:49,313] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-10-28T00:00:00+00:00 [running]> on host c3f731879890
[2022-08-06 21:50:49,411] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-10-28T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-28T00:00:00+00:00
[2022-08-06 21:50:49,412] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:50:49,413] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:50:49,429] {subprocess.py:85} INFO - Output:
[2022-08-06 21:50:49,431] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:50:49,431] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:50:49,464] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211028T000000, start_date=20220806T215049, end_date=20220806T215049
[2022-08-06 21:50:49,487] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:50:49,523] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:56:17,793] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-10-28T00:00:00+00:00 [queued]>
[2022-08-06 21:56:17,811] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-10-28T00:00:00+00:00 [queued]>
[2022-08-06 21:56:17,811] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:56:17,811] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:56:17,811] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:56:17,833] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-10-28 00:00:00+00:00
[2022-08-06 21:56:17,842] {standard_task_runner.py:52} INFO - Started process 949 to run task
[2022-08-06 21:56:17,847] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-10-28T00:00:00+00:00', '--job-id', '280', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmphtb_456t', '--error-file', '/tmp/tmpsgmqgvcr']
[2022-08-06 21:56:17,848] {standard_task_runner.py:80} INFO - Job 280: Subtask second_task
[2022-08-06 21:56:17,944] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-10-28T00:00:00+00:00 [running]> on host bc7e154a4fdd
[2022-08-06 21:56:18,083] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-10-28T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-28T00:00:00+00:00
[2022-08-06 21:56:18,084] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:56:18,085] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:56:18,101] {subprocess.py:85} INFO - Output:
[2022-08-06 21:56:18,102] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:56:18,103] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:56:18,149] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211028T000000, start_date=20220806T215617, end_date=20220806T215618
[2022-08-06 21:56:18,180] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:56:18,226] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:11:17,865] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-10-28T00:00:00+00:00 [queued]>
[2022-08-06 22:11:17,882] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-10-28T00:00:00+00:00 [queued]>
[2022-08-06 22:11:17,883] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:11:17,883] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:11:17,883] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:11:17,902] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-10-28 00:00:00+00:00
[2022-08-06 22:11:17,909] {standard_task_runner.py:52} INFO - Started process 953 to run task
[2022-08-06 22:11:17,913] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-10-28T00:00:00+00:00', '--job-id', '278', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpkk533nty', '--error-file', '/tmp/tmp6g99l8em']
[2022-08-06 22:11:17,913] {standard_task_runner.py:80} INFO - Job 278: Subtask second_task
[2022-08-06 22:11:17,988] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-10-28T00:00:00+00:00 [running]> on host 1250c3b659cd
[2022-08-06 22:11:18,073] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-10-28T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-28T00:00:00+00:00
[2022-08-06 22:11:18,074] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:11:18,075] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:11:18,092] {subprocess.py:85} INFO - Output:
[2022-08-06 22:11:18,094] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:11:18,094] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:11:18,133] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211028T000000, start_date=20220806T221117, end_date=20220806T221118
[2022-08-06 22:11:18,164] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:11:18,198] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:19:02,402] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-10-28T00:00:00+00:00 [queued]>
[2022-08-06 22:19:02,416] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-10-28T00:00:00+00:00 [queued]>
[2022-08-06 22:19:02,416] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:19:02,416] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:19:02,417] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:19:02,434] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-10-28 00:00:00+00:00
[2022-08-06 22:19:02,440] {standard_task_runner.py:52} INFO - Started process 957 to run task
[2022-08-06 22:19:02,443] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-10-28T00:00:00+00:00', '--job-id', '278', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpezrpkwii', '--error-file', '/tmp/tmp3d568rc4']
[2022-08-06 22:19:02,444] {standard_task_runner.py:80} INFO - Job 278: Subtask second_task
[2022-08-06 22:19:02,533] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-10-28T00:00:00+00:00 [running]> on host ed53bde1c44a
[2022-08-06 22:19:02,645] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-10-28T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-28T00:00:00+00:00
[2022-08-06 22:19:02,646] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:19:02,647] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:19:02,663] {subprocess.py:85} INFO - Output:
[2022-08-06 22:19:02,665] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:19:02,665] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:19:02,709] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211028T000000, start_date=20220806T221902, end_date=20220806T221902
[2022-08-06 22:19:02,735] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:19:02,788] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:22:41,623] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-10-28T00:00:00+00:00 [queued]>
[2022-08-06 22:22:41,646] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-10-28T00:00:00+00:00 [queued]>
[2022-08-06 22:22:41,646] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:22:41,646] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:22:41,646] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:22:41,669] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-10-28 00:00:00+00:00
[2022-08-06 22:22:41,677] {standard_task_runner.py:52} INFO - Started process 1010 to run task
[2022-08-06 22:22:41,681] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-10-28T00:00:00+00:00', '--job-id', '279', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpjaydma9y', '--error-file', '/tmp/tmpekhybn7s']
[2022-08-06 22:22:41,682] {standard_task_runner.py:80} INFO - Job 279: Subtask second_task
[2022-08-06 22:22:41,759] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-10-28T00:00:00+00:00 [running]> on host b215076695c3
[2022-08-06 22:22:41,856] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-10-28T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-28T00:00:00+00:00
[2022-08-06 22:22:41,857] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:22:41,858] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:22:41,872] {subprocess.py:85} INFO - Output:
[2022-08-06 22:22:41,873] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:22:41,874] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:22:41,918] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211028T000000, start_date=20220806T222241, end_date=20220806T222241
[2022-08-06 22:22:41,973] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:22:42,034] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
